modules/ROOT/partials/plugin-apis/inlinecss-apis.adoc

```diff
@@ -0,0 +1,57 @@
+A new API, `editor.getContent()` has been added to support the new `InlineCSS` plugin
+
+The API does not require configuration from the user server side, as the API’s function is to return a `string` around the HTML content specific that has `InlineCSS` styling applied.
+
+The API will do the following
+
+* Fire the 'InlineCSS' event
+* Get the editor's content by calling `editor.getContent()`
```

Comment:
@kemister85 The API is actually `editor.plugins.inlinecss.getContent();`. `editor.getContent()` is for generating just the HTML (not inlined CSS) and has been implemented for a long time.

Reply:
@Skylite73 Updated suggested changes regarding the API function ect.
@tiny-ben-tran The mention regarding what the API "under the hood" may require attention at another stage.